src/components/KSlideout/KSlideout.vue

```diff
@@ -224,7 +228,7 @@ onUnmounted(() => {
     position: fixed;
     right: 0;
     top: v-bind('offsetTopValue');
-    z-index: 9999;
+    z-index: v-bind(zIndex); // same value as z-index of slideout-container
```

Comment:
Since `slideout-backdrop` and `slideout-container` had the same `z-index` value, bounded both of them to the prop.

If only `z-index` of `slideout-container` was bound to the prop, it would allow someone to pass a `z-index` to `slideout-container` that could be less than the `z-index` of `slideout-backdrop`.

Another way to achieve this would be adding a validator to the `z-index` prop that would prevent passing a value lower than `9999` to it.

Reply:
this looks fine 👍🏼